pkg/pub_integration/lib/src/fake_pub_server_process.dart

```diff
@@ -34,6 +39,11 @@ class FakePubServerProcess {
     final process = await Process.start(
       'dart',
       [
+        if (collectCoverage) ...[
+          '--pause-isolates-on-exit',
+          '--enable-vm-service=$vmServicePort',
+          '--disable-service-auth-codes',
```

Comment:
This feels like a hack.. we'll have a hard time ripping this out... maybe we can add a line of documentation :)

Or we could pass an environment variable `PUB_DART_ARGUMENTS`, that way the `pkg/integration_test` don't need to know anything about coverage collection...

Reply:
Yeah, that could work out better...